src/pydiverse/pipedag/materialize/materializing_task.py

```diff
@@ -413,6 +413,9 @@ class AutoVersionType:
     To use it, you must specify
     :external+pl:doc:`polars.LazyFrame <reference/lazyframe/index>`
     as the task input type and only use LazyFrames inside your task.
+    For Polars, "Not inspecting the contents of the input tables" means
+    "not :external+pl:doc:`collecting <reference/lazyframe/api/polars.LazyFrame.collect>`
+    the input tables or any tables derived from them".
```

Comment:
There is a special case 
```suggestion
    the input tables or any tables derived from them". 
    Be careful not to collect any LazyFrames in such tasks (especially don't use collected data in conditional statements) unless a cache function is provided which indicates an input change for any data content changes. 
```